chart/templates/_helpers.yaml

```diff
@@ -49,6 +49,13 @@
       secretKeyRef:
         name: {{ .Release.Name }}-broker-url
         key: connection
+  {{- if .Values.flower.basicAuth}}
```

Comment:
It doesn't look very safe. Only the flower container should have access to this secret, and in particular the worker should NOT be able to access this secret.

Reply:
should I move this env to flower-deployment ?

Reply:
Yes. I think this is a good solution from a security point of view